src/commands/publish/route.rs

```diff
@@ -6,31 +8,31 @@ use cloudflare::framework::HttpApiClientConfig;
 
 use crate::http::{cf_v4_api_client, format_error};
 use crate::settings::global_user::GlobalUser;
-use crate::settings::toml::Route;
+use crate::settings::toml::{Route, Zoned};
 
 pub fn publish_routes(
     user: &GlobalUser,
-    routes: Vec<Route>,
-    zone_id: &String,
+    zoned_config: &Zoned,
 ) -> Result<Vec<RouteUploadResult>, failure::Error> {
     // For the moment, we'll just make this call once and make all our decisions based on the response.
     // There is a possibility of race conditions, but we just report back the results and allow the
     // user to decide how to procede.
-    let existing_routes = fetch_all(user, zone_id)?;
+    let existing_routes = fetch_all(user, &zoned_config.zone_id)?;
 
-    let deployed_routes = routes
+    let deployed_routes = zoned_config
+        .routes
         .iter()
-        .map(|route| deploy_route(user, zone_id, route, &existing_routes))
+        .map(|route| deploy_route(user, &zoned_config.zone_id, route, &existing_routes))
         .collect();
 
     Ok(deployed_routes)
 }
 
-fn fetch_all(user: &GlobalUser, zone_identifier: &String) -> Result<Vec<Route>, failure::Error> {
+fn fetch_all(user: &GlobalUser, zone_identifier: &str) -> Result<Vec<Route>, failure::Error> {
```

Comment:
nit: i think i'd name this `fetch_all_routes` but i could be persuaded since this is in `route.rs`

Reply:
yah it's in route, so i hope we're talking about routes, and it's a private function so will not be called from elsewhere.